recipes/leftbar/group_row/group_row.vue

```diff
@@ -110,3 +110,7 @@ export default {
   },
 };
 </script>
+
+<style lang="less" scoped>
+@import "../style/leftbar_row.less";
+</style>
```

Comment:
This is not necessary as this component is using `dt-recipe-general-row` as base, which already includes the stylings